test/e2e/main/main_test.go

```diff
@@ -274,13 +275,17 @@ func areOperatorLabelsPropagated(namespace, varName string, labels map[string]st
 	return true
 }
 
-// Run some functions for testing rights not covered by integration tests
+// TestRolesSynthetic runs some functions for testing rights not covered by integration tests
 func TestRolesSynthetic(t *testing.T) {
 	_, err := serviceAccountKube.Kubernetes.GetNodeHost(log, context.TODO())
 	tutils.ExpectNoError(err)
 
-	_, err = kube.FindStorageClass("not-present-storage-class", serviceAccountKube.Kubernetes.Client, context.TODO())
-	if !k8sErrors.IsNotFound(err) {
+	// Create a resource without passing any config
+	spec := tutils.DefaultSpec(testKube)
+
+	//_, err = kube.FindStorageClass("not-present-storage-class", serviceAccountKube.Kubernetes.Client, context.TODO())
+	if _, err := kube.LookupResource("not-present-storage-class", spec.Namespace, &storagev1.StorageClass{}, spec,
+		serviceAccountKube.Kubernetes.Client, log, r.eventRec, context.TODO()); !k8sErrors.IsNotFound(err) {
```

Comment:
hey @ryanemerson, after rebasing I started the refactoring of the `TestRolesSynthetic`, given that this PR is removing `FindStorageClass` as well. However, not sure here how to best approach the param `record.EventRecorder` from within the main e2e tests?

Reply:
In our tests instead of using `kube.LookupResource` it's simpler to just perform a `Get` to retrieve a resource using the kube client i.e. `client.Get(ctx, types.NamespacedName{Name: name}, storageClass)`


However, in this case we actually want to remove `TestRolesSynthetic` so we don't even have to do that :slightly_smiling_face:. You had done this before, so I think it was accidentally reintroduced when rebasing.

Reply:
🙂 kk! fixed now, thanks.